extensions/gc/GarbageCollectorMetrics.js

```diff
@@ -0,0 +1,40 @@
+const { ZenkoMetrics } = require('arsenal').metrics;
+const { getStatusLabel } = require('../utils/metricsUtil');
+
+const GC_LABEL_ORIGIN =  'origin';
+const GC_LABEL_OP = 'op';
+const GC_LABEL_STATUS = 'status';
+
+const gcS3Operations = ZenkoMetrics.createCounter({
+    name: 'gc_s3_operations',
+    help: 'Total number of S3 operations by the garbage collector processes',
+    labelNames: [
+        GC_LABEL_ORIGIN,
+        GC_LABEL_OP,
+        GC_LABEL_STATUS,
+    ],
+});
```

Comment:
GC may be called for reason other than cold storage expiration ; so we may want to also have some specific metric for restore expiration.

Reply:
indeed, i missed that (maybe after seeing the origin being hardcoded in other use cases...)